lib/cms/repo.ex

```diff
@@ -234,9 +234,11 @@ defmodule CMS.Repo do
   end
 
   defp params_to_string(params) when params == %{}, do: ""
+  defp params_to_string(nil), do: ""
```

Comment:
We should just make it `(_), do: ""` and put this after the string and map input types.